server/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -190,6 +191,12 @@ public String toString() {
         return Strings.toString(this);
     }
 
+    // TODO remove this and implement default in every QueryBuilder
```

Comment:
in every AggregationBuilder? I would also be fine removing these TODOs given that we are planning to address them next.

Reply:
Removing those comments then